README.md

```diff
@@ -88,12 +88,12 @@ Ruby unit and feature tests:
 bundle exec rspec
 ```
 
-### Karma
+### Jest
 
 JavaScript unit tests:
 
 ```bash
-yarn karma start --single-run
+yarn test
```

Comment:
Maybe add an example of how to run a single spec file? Seems to be a common question with testing.